src/components/CodeEditor.tsx

```diff
@@ -276,7 +306,7 @@ export const CodeEditor = ({
     return () => {
       view.destroy()
     }
-  }, [code !== "", !isStreaming])
+  }, [!isStreaming, currentFile])
```

Comment:
i think this re-introduces a bug i fixed. We want to trigger the `useEffect` when the code goes from "" -> anything because that means that the code has loaded

```suggestion
  }, [!isStreaming, currentFile, code !== ""])
```